data/containers/autoyast_containers.xml.ep

```diff
@@ -10,7 +10,7 @@
     <reg_code>{{SCC_REGCODE}}</reg_code>
     <reg_server>{{SCC_URL}}</reg_server>
     <addons config:type="list">
-      % if ($get_var->('VERSION') =~ /^15/) {
+      % if ($get_var->('VERSION') =~ /^15|^16/) {
```

Comment:
IIUC, there are still no addons/modules in 16.0

Reply:
ok, but I updated all the possible, if not exists, none added: it  shouldn't break anything. Isn't it?

Reply:
SCC will complain and fail

Reply:
just updated